src/main/Config.cpp

```diff
@@ -1151,6 +1151,12 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
                      CATCHUP_RECENT =
                          readInt<uint32_t>(item, 0, UINT32_MAX - 1);
                  }},
+#ifdef BUILD_TESTS
+                {"CATCHUP_SKIP_KNOWN_RESULTS_FOR_TESTING",
+                 [&]() {
+                     CATCHUP_SKIP_KNOWN_RESULTS_FOR_TESTING = readBool(item);
```

Comment:
I missed this earlier - a default value of `false` should be set in `Config::Config`.